src/main/java/nextstep/ladder2/player/Players.java

```diff
@@ -0,0 +1,73 @@
+package nextstep.ladder2.player;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class Players {
+    private List<Player> players = new ArrayList<>();
+    private static final Player NONE = Player.of("NONE", Integer.MAX_VALUE-1, Integer.MAX_VALUE);
+    public Players(List<Player> players) {
+        this.players = players;
+    }
+
+    private Players(String... names) {
+        this.players = makeGroup(names);
+    }
+
+    public static Players of(String s) {
+        String[] names = s.split(",");
+        return new Players(names);
+    }
+
+    private List<Player> makeGroup(String[] names) {
+        List<Player> group = new ArrayList<>();
+        AtomicInteger index = new AtomicInteger();
+        Arrays.stream(names)
+                .forEach(
+                        name -> group.add(
+                                Player.of(name, index.getAndIncrement(), names.length-1)
+                        )
+                );
+        return group;
+    }
+
+    public List<String> names(){
+        return players.stream()
+                .map(Player::name)
+                .collect(Collectors.toList());
+    }
+
+    public List<Integer> indexes(){
+        return players.stream()
+                .map(Player::index)
+                .collect(Collectors.toList());
+    }
+
+    public void move(List<Direction> directions) {
+        AtomicInteger index = new AtomicInteger();
+        directions.stream()
+                .forEach(dir -> {
+                    players.get(index.getAndIncrement()).move(dir);
+                });
```

Comment:
```suggestion
        directions .forEach(dir -> players.get(index.getAndIncrement()).move(dir));
```
List는 바로 forEach를 사용해줘도 된답니다!